src/views/home/ContentDetail.vue

```diff
@@ -41,7 +41,8 @@
             "
           >
             (less)</a
-          >
+          ><br>
+          <button @click="goToReview" id="newreview">New Review</button><br>
```

Comment:
```suggestion
          <button @click="createReview" id="new-review">New Review</button><br>
```
`goToReview` isimlendirmesi ilk bakışta inceleme detayına gidilecekmiş gibi bir izlenim veriyor